README.md

```diff
@@ -1,2 +1,37 @@
-# mobile-ui-sdks
-Mobile SDKs for authentication and building customizable login UIs for Asgardeo &amp; WSO2 Identity Server.
+<h1 align="center" style="color: #343a40;margin: 20px 0;">
+  <p align="center"><span style="color: #47EBD8">Mobile SDKs</span> for <a href="https://wso2.com" style="color:#ff6923">Asgardeo</a></p>
+</h1>
+
+<div align="center">
+  <a href="https://stackoverflow.com/questions/tagged/wso2is"><img src="https://img.shields.io/badge/Ask%20for%20help%20on-Stackoverflow-orange" alt="Stackoverflow"></a>
+  <a href="https://discord.gg/wso2"><img src="https://img.shields.io/badge/Join%20us%20on-Discord-%23e01563.svg" alt="Discord"></a>
+  <a href="./LICENSE"><img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="License"></a>
+</div>
+
+<br>
+
+This repository holds the mobile SDKs to support the integreation of your mobile application with the WSO2's identity and access management products, WSO2 Identity Server and Asgardeo
```

Comment:
Let's remove the Identity server references.

Reply:
Fixed with https://github.com/asgardeo/mobile-ui-sdks/pull/1/commits/67ea36cc9fef864e59221f887574afe50546eea3